modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc

```diff
@@ -0,0 +1,73 @@
+:description: How to use Cypher to manage property-based access control on graphs.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE regularUsers;
+----
+////
+
+[role=enterprise-edition aura-db-enterprise]
+[[property-based-access-control]]
+= Property-based access control
+
+It is possible to create read privileges that are based on properties of nodes.
+Each property-based privilege can only be restricted by a single property.
+To specify the property/value conditions of the read privilege the `pattern` syntax described below is used,
+for more information about read privilege syntax see xref:authentication-authorization/privileges-reads.adoc[read privilege] page.
+
+Adding property-based access control can incur a significant performance overhead in certain scenarios,
+see xref:authentication-authorization/limitations.adoc#property-based-access-control-limitations[Limitations] for more detailed information.
+To reduce the performance impact, we recommend using Block Storage as this is better optimised for the kind of read required for the resolution of property-based privileges.
+
+Factors which will exacerbate the performance impact of having property rules are:
+
+1. The number of properties on the nodes concerned (more properties = greater performance impact)
+2. The number of property-based privileges (more property-based privileges = greater performance impact).
+3. The type of the privilege: `TRAVERSE` property-based privileges have a greater performance impact than `READ` property-based privileges.
+
+For performance-critical scenarios it is recommended to design  xref:authentication-authorization/privileges-reads.adoc[privileges based upon labels].
+
+Pattern syntax:
+[source, syntax, role="noheader"]
+----
+([var][:label["|" ...]] "{" property: value "}")
+| (var[:label["|" ...]]) WHERE [NOT] var.property { = value | <> value | IS NULL | IS NOT NULL }
+| (var[:label["|" ...]] WHERE [NOT] var.property { = value | <> value | IS NULL | IS NOT NULL } )
+----
+[NOTE]
+====
+For more details about the syntax descriptions, see xref:database-administration/syntax.adoc[].
+====
+This pattern syntax are used in the xref:authentication-authorization/privileges-reads.adoc[read privileges] as this:
```

Comment:
We try to avoid using words like "as this", "like this", "this is how you can do it", etc.:
 
```suggestion
You can use this pattern syntax for defining read privileges as follows:
```